landscape.yml

```diff
@@ -2973,12 +2973,23 @@ landscape:
               incubating: '2018-12-11'
               graduated: '2020-11-24'
               dev_stats_url: https://etcd.devstats.cncf.io/
-              artwork_url: https://github.com/cncf/artwork/blob/master/examples/incubating.md#etcd-logos
+              artwork_url: https://github.com/cncf/artwork/blob/master/examples/graduated.md#etcd-logos
               stack_overflow_url: https://stackoverflow.com/questions/tagged/etcd
               mailing_list_url: https://groups.google.com/forum/?hl=en#!forum/etcd-dev
-              slack_url: https://freenode.net/
+              slack_url: http://slack.k8s.io/
               chat_channel: '#etcd'
               clomonitor_name: etcd
+              summary_personas: Architects, Platform Engineers, SREs
+              summary_tags: distributed, key-value, consistent, datastore, reliability
+              summary_use_case: >
+                Etcd is a distributed, reliable key-value store for the most critical data of a distributed system.
+
+                By using etcd, developers can ensure that their applications have access to up-to-date configuration data, even as they scale up or down, and can maintain consistency, fault tolerance and coordination across multiple instances of the application.
+              summary_business_use_case: >
+                Etcd can be used by businesses to manage the configuration data and coordination needs of their applications, ensuring consistent and reliable performance at scale.
+              summary_release_rate: Every 1-3 months
```

Comment:
Not sure about release rate. I don't think we should count patch releases. Most other projects in this file mention that they count major or minor releases.

To note
* https://github.com/etcd-io/etcd/releases/tag/v3.5.0 was June 2021
* https://github.com/etcd-io/etcd/releases/tag/v3.4.0 was Aug 2019

So we do a minor release once per 2 years. Assuming that we will do v3.6 this year, which is hard to guarantee as there was zero progress from last year when I proposed it.

Reply:
How about: `Major or Minor release approximately every 2 years, and patch release every 1-3 months` ? 

cc @mitake @ptabor @serathius @spzala